ros/src/actuation/vehicles/packages/as/nodes/as_interface/as_interface.cpp

```diff
@@ -51,142 +49,114 @@ void PacmodInterface::initForROS()
   private_nh_.param<double>("acceleration_limit", acceleration_limit_, 3.0);
   private_nh_.param<double>("deceleration_limit", deceleration_limit_, 3.0);
   private_nh_.param<double>("max_curvature_rate", max_curvature_rate_, 0.75);
-  private_nh_.param<bool>("use_curvature_cmd_", use_curvature_cmd_, true);
+  private_nh_.param<bool>("use_curvature_cmd", use_curvature_cmd_, true);
+  private_nh_.param<bool>("use_timer_publisher", use_timer_publisher_, false);
+  private_nh_.param<double>("publish_frequency", publish_frequency_, 10.0);
 
   // setup subscriber
-  if(!use_curvature_cmd_){
-	  twist_cmd_sub_    = nh_.subscribe("twist_cmd", 1, &PacmodInterface::callbackFromTwistCmd, this);
-  }else{
-	  curvature_cmd_sub_ = nh_.subscribe("curvature_cmd", 1, &PacmodInterface::callbackFromCurvatureCmd, this);
+  if (!use_curvature_cmd_)
+  {
+    twist_cmd_sub_ = nh_.subscribe("twist_cmd", 1, &PacmodInterface::callbackFromTwistCmd, this);
+  }
+  else
+  {
+    curvature_cmd_sub_ = nh_.subscribe("curvature_cmd", 1, &PacmodInterface::callbackFromCurvatureCmd, this);
   }
 
   control_mode_sub_ = nh_.subscribe("/as/control_mode", 1, &PacmodInterface::callbackFromControlMode, this);
-  speed_sub_        = nh_.subscribe("/vehicle/steering_report", 1, &PacmodInterface::callbackFromSteeringReport, this);
+  speed_sub_ = nh_.subscribe("/vehicle/steering_report", 1, &PacmodInterface::callbackFromSteeringReport, this);
```

Comment:
this topic does not exist for our version of pacmod. should we get rid of it?

Reply:
I changed this topic to "/as/velocity_accel", thx.
Please use "as_velocity" in vel_pose_connect node.